src/OneExplorer/ConfigObject.ts

```diff
@@ -23,9 +23,10 @@ import {RealPath} from '../Utils/Helpers';
 import {Logger} from '../Utils/Logger';
 
 /**
- * 'Artifact' means includes
- *   - existing files to run ONE config (base model)
- *   - result files after running ONE config (derived models, )
+ * 'Artifact'
+ * The collective term is and inclusive, it includes two types of files:
+ * (1) Pre-existing files to run ONE config (base model)
+ * (2) Result files after running ONE config (derived models, )
```

Comment:
```suggestion
 * (2) Result files after running ONE config (derived models)
```

Reply:
@hyunsik-yoon Actually, this change is part of #1058 PR. PTAL :-D
https://github.com/Samsung/ONE-vscode/pull/1058/files#r924070496